src/main.cpp

```diff
@@ -8,10 +8,77 @@
 #include <iostream>
 #include "window.hpp"
 
+#include "../include/model.hpp"
```

Comment:
You shouldn't need to use a relative include path like this. Also file names should start with a capital letter

Reply:
Yeah but i think somethong miss in CMakeList because i can't delete the path of the file